src/tests/integration/test_foundry_prove.py

```diff
@@ -633,3 +634,38 @@ def test_deployment_summary(
         TEST_DATA_DIR / 'foundry' / 'src' / 'DeploymentStateCode.sol',
         update=update_expected_output,
     )
+
+
+def test_foundry_xml_report(
```

Comment:
@ehildenb thank you for your suggestion! I just added the test. 

> Let's add a test to `test_foundry_prove.py` which exercises the new functionality.
> 
> * Pick a list of 3-5 short-running proofs. Run them in a batch with `foundry_prove(...)`.
> * Generate the XML report.
> * Re-load the XML report with a junit-style reader, to make sure it's understandable as an XML test report.

Reply:
The test looks great, thanks @lisandrasilva! But it looks like the legacy backend job timed out on the previous run: https://github.com/runtimeverification/kontrol/actions/runs/8138402331/job/22251131183?pr=377. I'd say we can skip `test_foundry_xml_report` on legacy backend runs, e.g., you can just add
```
    if no_use_booster:
        pytest.skip()
```
to the test (you'll also need to add `no_use_booster: bool,` to the function signature).